contents/swift/welcome-to-swift.md

```diff
@@ -218,9 +261,9 @@ barcode2.printCode() // prints "QR code: bar."
 
 Also, enums with associated values is not supported in languages such as [Java](https://stackoverflow.com/questions/30044334/how-can-i-create-a-java-enum-with-associated-values-like-swift-enum), and using a workaround to implement enums with associated values results in code verbosity. Take a look at [Swift's documentation on Enums](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Enumerations.html) for more information about enums.
 
-## Protocol Oriented Programming
+### Benefit: Protocol Oriented Programming
```

Comment:
I'm a little hesistant about calling this a benefit. Your other benefit headings are phrased in such a way that they are benefits (for e.g. "Declare variables easily", "Handle Special Values Meaningfully"), but I wouldn't call "Protocol Oriented Programming" a benefit. Maybe "Supports Protocol Oriented Programming"?

Reply:
Good idea! I have also apply it in some other `Benefits` titles